### ISSUE TITLE
Uncaught TypeError: Class extends value undefined is not a constructor or null : asStream.js:5:46

### ISSUE THREAD

**Issue by ruben-certhis** [torusresearch/torus-embed#497]
**Describe the bug**
Hi , when i implement "@toruslabs/torus-embed" i got error 

My code :
```
import Torus from "@toruslabs/torus-embed";
import Web3 from "web3";
const torus = new Torus();
```

Error :

asStream.js:5 Uncaught TypeError: Class extends value undefined is not a constructor or null
    at eval (asStream.js:5:46)
    at ./node_modules/@metamask/obs-store/dist/asStream.js (compile_app.js:63:1)
    at __webpack_require__ (compile_app.js:1304:42)
    at eval (index.js:13:14)
    at ./node_modules/@metamask/obs-store/dist/index.js (compile_app.js:74:1)
    at __webpack_require__ (compile_app.js:1304:42)
    at eval (torus.esm.js:19:77)
    at ./node_modules/@toruslabs/torus-embed/dist/torus.esm.js (compile_app.js:140:1)
    at __webpack_require__ (compile_app.js:1304:42)
    at eval (app.js:2:80)

**Comment by Vobble-fun** [comment]
@ruben-certhis  I'm also facing the same issue. Any solutions on this?

**Comment by cozmbatman** [comment]
@ruben-certhis @Vobble-fun did you guys find any resolution for this?

**Comment by ruben-certhis** [comment]
We created our own wallet ....

**Comment by chaitanyapotti** [comment]
This is fixed now in 2.0.0
